scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# This file is part of qutebrowser.
+
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+'''Tool to import browser history data from other browsers. Although, safari
+support is still on the way.'''
+
+
+import argparse
+import sqlite3
+import sys
+
+
+def parser():
+    """Parse command line arguments."""
+    description = 'This program is meant to extract browser history from your'\
+                  'previous browser and import them into qutebrowser.'
```

Comment:
Please put parentheses around the whole string, then you can break lines without needing to use `\`.

Reply:
You also might want to use double-quotes everywhere here, then you can use `'` without needing to escape it.